src/main/java/step1/Calculator.java

```diff
@@ -0,0 +1,30 @@
+package step1;
+
+public class Calculator {
+
+    public int calculate(String[] inputValue) {
+        int result = 0;
+        for(String number: inputValue){
+            result += validateNumber(number);
+        }
+
+        return result;
+    }
+
+    public int validateNumber(String number){
```

Comment:
파라미터와 블록 시작의 괄호 사이에 한칸 띄어주시는게 관례이오니 참고 부탁드리겠습니다 :)
다른 메서드들도 반영해주세요 !
```suggestion
    public int validateNumber(String number) {
```